Update job page for spring 2023 hiring (#38)
umn-asr/umn-asr.github.io: jobs.markdown

#### Values

We build with code, but our foundations are these values

- Diversity: A mix of experiences, opinions and approaches makes all our work better
- Best Practices: Use and refine techniques like Pair Programming and Test Driven Development that result in better code
- Continuous Learning: We dedicate time each week for learning new tools, languages and techniques
- Mentoring: Take what you have learned and return it to the community

#### Benefits

The Office of Human Resources has a [guide to the University's benefits](https://humanresources.umn.edu/benefits) including the [Regent's Scholarship](https://humanresources.umn.edu/benefits/tuition-benefits), [Faculty Retirement Plan](http://humanresources.umn.edu/employee-benefits/retirement#anchor-auto), [Leave](https://humanresources.umn.edu/benefits/vacations-and-leaves), and [Medical](https://humanresources.umn.edu/employee-benefits/medical).

For this job we have funding for a salary range of 65,000 to 80,000.

### How to Apply

> Ready to apply? Great! Start at [the job posting](https://z.umn.edu/asr_dev2_external).

Reviewer: ```suggestion
Ready to apply? Great! Start at [the job posting](https://z.umn.edu/asr_ror_job_external)).
```
This is current link, correct?

Before you start, you should have:

- A resume
   - Formatted or a plain text document, either is fine
- A cover letter
  - Tell us what excites you about this job, work you've done and where you want your career to go
- Names and contact information for three references
  - Mentors, bosses, co-workers, people you study with or talk to at meetups

You can take a break at any point during the application process. Click the **Save Draft** button and you can return to your application later. When you log back into the job application system, click **My Activities** and then click on the job title **Web Application Developer**

If you have any questions while applying, look at the [Using the Job Application System](http://humanresources.umn.edu/find-job/using-employment-system) page.

### Interviews

We use three interviews.

1. A brief conversation over Zoom about your experience and interest in the job

> 1. A panel interview with members of the Application Development team

Reviewer: ```suggestion
1. A panel interview with members of the Application Development team and one of our stakeholders
```
I don't think it would be too concerning for a candidate when they see someone who is not on our team in the interview, but it is still probably good to be clear about it

1. Code reading and discussion with Application Development team members

Oh, and we won't ask you to code anything on a whiteboard.

All interviews will be remote.

### Answers to Some Common Questions

**Is this job remote or in person?**

Our work location will be flexible. You can work on campus, at home, or a mix.

**Do I need to be in Minnesota?**

> You should be within commuting distance of Minneapolis, MN.

Reviewer: Based upon what Carin said, it sounds like this is a little more flexible, maybe something like:
```suggestion
You should be able to travel to Minneapolis, MN a few days per month.
```
